e2e_example/test/common/utils.dart

```diff
@@ -177,23 +177,28 @@ Future<String> nextStdOutLine(String message) =>
     _stdOutLines.firstWhere((line) => line.contains(message)) as Future<String>;
 
 /// Runs tests using the manual build script.
-Future<ProcessResult> _runManualTests({bool usePrecompiled}) {
+Future<ProcessResult> _runManualTests(
+    {bool usePrecompiled, List<String> args}) {
```

Comment:
nit: `buildArgs` for consistency/clarity (same below)

Reply:
Done